webapp-django/crashstats/authentication/migration.py

```diff
@@ -1,65 +0,0 @@
-from __future__ import print_function
```

Comment:
We don't need this anymore--it's for migration users from their email alias to their LDAP account.